src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php

```diff
@@ -1028,4 +1030,23 @@ public function switchToIFrame($locator = null)
             $this->webDriver->switchTo()->frame($els[0]);
         }
     }
+
+    /**
+     * Invoke Codeption pause()
+     *
+     * @param boolean $pauseOnFail
+     * @return void
+     */
+    public function pause($pauseOnFail = false)
+    {
+        if (!\Codeception\Util\Debug::isEnabled()) {
+            return;
+        }
+
+        if ($pauseOnFail) {
+            print(PHP_EOL . PHP_EOL . "Failure encountered. Pausing execution..." . PHP_EOL . PHP_EOL . PHP_EOL);
```

Comment:
Message looks good, may be too spaced out due to PHP_EOLs. I added both formats below, prefer the second one. See if you want to change it to the 2nd format.
```
[assertLoggedIn] AssertAdminSuccessLoginActionGroup
  [waitForAdminAccountTextVisible] wait for element visible ".page-header .admin-user-account-text",30


Failure encountered. Pausing execution...


  Execution PAUSED, starting interactive shell...
```
Instead 

```
[assertLoggedIn] AssertAdminSuccessLoginActionGroup
  [waitForAdminAccountTextVisible] wait for element visible ".page-header .admin-user-account-text",30
Failure encountered. Pausing execution...
  Execution PAUSED, starting interactive shell...
```

Reply:
Ok, I will revert this change

Reply:
Reverted. I actually prefer the former.